ledger/participant-integration-api/src/main/scala/platform/indexer/JdbcIndexer.scala

```diff
@@ -133,6 +232,8 @@ private[indexer] class JdbcIndexer private[indexer] (
   private def handleStateUpdate(offset: Offset, update: Update): Future[Unit] = {
     lastReceivedRecordTime = update.recordTime.toInstant.toEpochMilli
 
+    logger.trace(s"Update (${update.getClass.getSimpleName})")
```

Comment:
Seems like that class name could also go into the context.

Reply:
I'm a bit hesitant on this one. It uses reflection, which can be quite a hit. And while `trace` makes sure the the string interpolation is evaluated lazily, the context creation is not, which is probably something we should fix. Still, it's probably not an actual bottleneck, so I'll gladly add it to the context and open a ticket to track this conversation.

Reply:
Even better, every `Update` has a description, I'll use that as a message. Still, I think it would be relevant to see if it could make sense to make context construction lazy.

Reply:
That makes sense!

Reply:
Addressed by https://github.com/digital-asset/daml/pull/6924/commits/c15aed4bb7897d36f227a71b2d372e9670750bf4

Reply:
Opened https://github.com/digital-asset/daml/issues/6944